### PR TITLE
Add integrationTestReport task

### PR DESCRIPTION
**Pull request by kant2002**
This task will run integrationTest task and save all test reports in the
`$reportDir/integrationTestReports` folder

This was requested in #832 by @adamopolous

### CODE REVIEW

File: gradle/root/test.gradle [hunk 1]
```diff
@@ -327,10 +327,23 @@ task parallelCombinedTestReport(type: TestReport) { t ->
...
 
+/*********************************************************************************
+ * INTEGRATION TEST REPORT
+ *
+ * Summary:	Runs all integration tests and generates a single report.
+ *
+ *********************************************************************************/
+task integrationTestReport(type: TestReport) { t ->
+	group "test"
+    description "Run unit integrtion tests and save HTML report."
```
Comment (adamopolous) [inline gradle/root/test.gradle]: Small misspelling fix, and remove "unit":

"Run integration tests and save...."

**Comment by adamopolous** [comment]
Thanks, @kant2002 !  

I just made a comment about a simple spelling/grammar fix. After that it'll be good to go.

**Comment by kant2002** [comment]
@adamopolous Please take a look.

**Comment by adamopolous** [comment]
Excellent - it's good to go

**Pull request merged**